components/Tabs/src/MDCTabBar.h

```diff
@@ -55,8 +55,8 @@ IB_DESIGNABLE
  The bar determines the newly-selected item using the following logic:
  * Reselect the previously-selected item if it's still present in `items` after the update.
  * If there was no selection previously or if the old selected item is gone, select the first item.
-   Clients that need empty selection to be preserved across updates to `items` must manually reset
-   selectedItem to nil after the update.
+ Clients that need empty selection to be preserved across updates to `items` must manually reset
```

Comment:
Nit: Please revert this file - The old formatting looked better to indicates that line 58 was a continuation of the  list item.

Reply:
oops. I didn't do it intentionally.

Addressed in 7f1c93e58fc931a53c7cc7bfd5debf7b157bd6aa